plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -227,7 +221,7 @@ func (s *SQLServer) Gather(acc telegraf.Accumulator) error {
 		}
 	}
 
-	if len(s.Servers) == 0 && s.DatabaseType == "SQLServer" && !s.IgnoreDefaultServer {
+	if len(s.Servers) == 0 {
 		s.Servers = append(s.Servers, defaultServer)
 	}
```

Comment:
This needs to move to the init() function to be override-able. You won't need the conditional there, so just s.Servers = []string{defaultServer}

Reply:
@ssoroka - That's an existing code, I can address it when I try to address the default server aspect in a separate PR. This PR is only for the HA/DR queries now.

Reply:
Alright. Please do make another PR for it, as it's currently busted, as you've noticed.

Reply:
Sure, will do that. Thank you!